spec/models/journal_notification_mailer_spec.rb

```diff
@@ -94,14 +87,14 @@ def call_listener
         end
 
         it 'sends no notification' do
-          expect(Delayed::Job).not_to receive(:enqueue)
+          expect(EnqueueWorkPackageNotificationJob).not_to receive(:set)
```

Comment:
While it should always be called in conjunction with `set` I would feel more at ease if we also explicitly made sure that `perform_later` is not called here and in the following cases.